lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@muzaffaryousaf @andy-armstrong 
Hi,

I am sorry about this issue.
I realized what I missing.

For this case, If we want to use custum preview domain,
we have to change 3 variables.

1.
```
"PREVIEW_LMS_BASE": "preview2.hoge.com"
```

2.
```
PREVIEW_DOMAIN = 'preview2'
```

3.
```
"HOSTNAME_MODULESTORE_DEFAULT_MAPPINGS": {
    "preview2\\.": "draft-preferred"
},
```

I missed point at 3.

But I want to ask one more things about preview domain(including preview.edx.org).
We planing to use preview function as mentioned.
But we are concerned about leakage of unpublished contents from preview domain.
Why edX keep to publish and accessible this URL to everyone?

Preview domain should publish to authorized user only, is not?
Or should use "Hide from students" option when we create every unpublished contents?

So what's your take on this?

Thanks,
Shohei